src/Common/Core/Model.php

```diff
@@ -336,4 +337,13 @@ public static function getSession(): SessionInterface
 
         return $session;
     }
+
+    private static function getMockSession(): Session
+    {
+        if (!self::getContainer()->has('fork.mock.session')) {
```

Comment:
Could you at least explain what this does in a comment or the PR?

Reply:
added in 52bee3e5a